src/Omni/Helper/OrderHelper.php

```diff
@@ -117,21 +117,31 @@ public function prepareOrder(Model\Order $order, $oneListCalculateResponse)
             $storeId       = $oneListCalculateResponse->getStoreId();
             $cardId        = $oneListCalculateResponse->getCardId();
             $customerEmail = $order->getCustomerEmail();
-            $customerName  = $order->getShippingAddress()->getFirstname() .
-                ' ' . $order->getShippingAddress()->getLastname();
+            $customerName  = $order->getBillingAddress()->getFirstname() .
++                ' ' . $order->getBillingAddress()->getLastname();
```

Comment:
This extra `+` character might introduce problems, no?

https://github.com/lsretailomni/lsmag-two/blob/11d71db949b6c35d1f5d1160463fe915df8abbf0/src/Omni/Helper/OrderHelper.php#L121

@duckchip, @lsretailomni

Reply:
good catch. This will introduce errors